docs/editing_guidelines.md

```diff
@@ -0,0 +1,138 @@
+3. Reporting editorial decisions (or decisions in waiting) about the term. This includes providing a reason for obsoleting a term, or letting users and curators know that the term may be merged or split in the future, e.g. when enough evidence for the merge/split will be available.
+
+Try to be consistent in how you phrase the various types of comments. For example:
+
+* when giving a reason for obsoletion, use “Obsoleted as ...”;
+* when indicating a potential future merge, use “Possible equivalence with {other term} ...”.
+
+
+## Synonyms
+Extensive addition of synonyms help searching. Add as many synonyms as you like.
+
+Guidelines on the type of synonyms:
+
+1. Use an _exact_ synonym when the name and the synonym can be used interchangeably, at least in a given context. This includes the case where the synonym is an abbreviation that, in a broader context, could have a different meaning. For example, _DPM_  can be an exact synonym of _dorsal pharyngeal muscle cell_, even though “DPM” could have many more meanings generally, because in the context where “dorsal pharyngeal muscle cell” is used, “DPM” is commonly used interchangeably to refer to these cells.
+2. There is no expectation that exact synonyms should be unique across the entire ontology. For example, in the _Drosophila_ Anatomy Ontology _DPM_ is also an exact synonym of _mushroom body dorsal paired medial cell_.
+3. Be mindful of the “directionality” of the _narrow_ and _broad_ types of synonyms. They qualify the _synonym_, not the original term. For example, saying that _dmd5_ is a narrow synonym of _abdominal dorsal multidendritic neuron_ means that “dmd5“ refers to a narrower concept than _abdominal dorsal multidendritic neuron_, not the other way around.
+4. The _related_ type may be used for other cases where the overlap between the synonym and the term name may be unclear, but you want people to find the term when they are searching for the synonym.
+
+
+## Considerations on style
+The following considerations apply both to all human-readable fields (names, textual definitions, comments, synonyms).
+
+* **US english or British english?** Where there are differences in the accepted spelling between english and US usage, use the US form. British english variants of the labels may be added as synonyms (e.g., a term labelled “epithelial cell of esophagus” may have “epithelial cell of oesophagus” as an exact synonym).
+* **Use of jargon.** The aim of the ontology is to provide useable descriptions and links to the reader. Consequently, try to avoid obscure jargon or pretentious latin/greek, especially where widely understood, plain alternatives exist. Where it will aid searching, such terms may be added to synonyms and/or as asides in the “gloss” part of definitions.
```

Comment:
```suggestion
* **Use of jargon.** The aim of the ontology is to provide useable descriptions and links to the reader. Consequently, try to avoid obscure jargon or pretentious Latin/Greek, especially where widely understood, plain alternatives exist. Where it will aid searching, such terms may be added to synonyms and/or as asides in the “gloss” part of definitions.
```